drivers/interrupt_controller/Kconfig.plic

```diff
@@ -10,3 +10,14 @@ config PLIC
 	help
 	  Platform Level Interrupt Controller provides support
 	  for external interrupt lines defined by the RISC-V SoC.
+
+if PLIC
+
+config PLIC_DEBUG
+	bool "PLIC debugging shell commands"
+	depends on SHELL
+	help
+	  Enable additional shell commands useful for debugging.
+	  Caution: This will use a lot of RAM (IRQs * sizeof(size_t)).
```

Comment:
If that's the case, do we want to print a warning from CMake if this symbol is enabled, so that we're sure the user is informed about this?